Dockerfile

```diff
@@ -1,4 +1,4 @@
-FROM node
+FROM node:12
```

Comment:
Is there a reason we went with version 12 instead of `latest` or `13`?

Reply:
13 isn't a LTS (long term supported release). Latest is a bit ephemeral; it doesn't always point to the latest build. More here: https://medium.com/@mccode/the-misunderstood-docker-tag-latest-af3babfd6375

Here's some info on Node releases and LTS: https://nodejs.org/en/about/releases/

Reply:
Ahh got it. Fair enough. I just wanted to make sure.